src/utils.ts

```diff
@@ -540,6 +564,24 @@ export class Utils {
     (target || e.target).dispatchEvent(simulatedEvent);
   }
 
+  public static getScaleForElement(element: HTMLElement) {

```

Comment:
(el: HTMLElement) then you don't need new local val.

Reply:
Overriding a parameter is clearly not a way to go. It would make the code much less readable